src/migration/1687960180394-PostFlags.ts

```diff
@@ -0,0 +1,22 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class PostFlags1687960180394 implements MigrationInterface {
+    name = 'PostFlags1687960180394'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
@capJavert shouldn't we set here the flags based on existing values or is it part of another migration?

Reply:
That was my main concern above, you mentioned they should be the same as existing ones right?
Would suggest to do it in the move-over PR though in that case, while migrating the values to flags

Just wanted to make sure I understood correctly.

Reply:
@idoshamun yeah, that one is done in a separate PR, that one also keeps them up to date. https://github.com/dailydotdev/daily-api/pull/1333

Reply:
We should consider including in this PR the implementation that updates the flags instead of separating the creation of the column and updating the column value.
It's not the actually move over but just making sure they are always up to date. Wdyt?

Reply:
The PR I mentioned actually targets merge into this one so feel free to review https://github.com/dailydotdev/daily-api/pull/1333 and then we can merge all in one go to `main`